### ISSUE TITLE
Q: DSPL point to person

### ISSUE THREAD

**Issue by joannetruong** [RoboCupAtHome/RuleBook#670]
For DSPL, would it be sufficient for the robot to pan its head towards someone that needs to be "pointed to" (i.e. when introducing them)?

**Comment by kyordhel** [comment]
# TL; DR: The robot must be facing towards the person

All interactions should be as natural as possible, leaving no room for ambiguities. Most robots have a quite wide FoV. The person’s face can be perfectly recognised in a corner, causing the impression that the robot is not looking at the person who is speaking to, but to a ghost standing aside. This can create awkward situations.   

 Although to the robot is clear who it is speaking with, for the person (and the referee) this may not be the case. Therefore panning must be addressed with care. Furthermore, aligning the robot towards the person gives a more polite impression than just rotating the head (uncanny in angles outside [-60°, 60°]).

If the referee thinks that the robot is interacting with a plant or other kind of furniture, it may not consider the interaction as valid (false positive).
